static/app/utils/number/formatMetricUsingUnit.tsx

```diff
@@ -59,60 +59,9 @@ function formatDuration(seconds: number): string {
 
 // The metric units that we have support for in the UI
 // others will still be displayed, but will not have any effect on formatting
-export const formattingSupportedMetricUnits = [
```

Comment:
we only had this `const` to compute the type, so now I’m computing the type from the keys used in `METRIC_UNIT_TO_SHORT`.